tests/models/vipllava/test_modeling_vipllava.py

```diff
@@ -185,6 +185,14 @@ def test_training_gradient_checkpointing_use_reentrant(self):
     def test_training_gradient_checkpointing_use_reentrant_false(self):
         pass
 
+    @unittest.skip(reason="Compile not yet supported because in LLava models")
```

Comment:
The sentence here doesn't parse - it reads as though the final bit is cut-off. 

Maybe this? 

```suggestion
    @unittest.skip(reason="Compile not yet supported because it is not yet supported in LLava")
```